aspnet/tutorials/web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.IO;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using TodoApi.Models;
+using Swashbuckle.Swagger.Model;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace TodoApi
+{
+    public class Startup
+    {
+
+        #region snippet_Configure
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add framework services.
+            services.AddMvc();
+
+            services.AddLogging();
+
+            // Add our repository type
+            services.AddSingleton<ITodoRepository, TodoRepository>();
+
+            // Inject an implementation of ISwaggerProvider with defaulted settings applied
+            services.AddSwaggerGen();
+
+            // Add the detail Info for the API
+            services.ConfigureSwaggerGen(options =>
+            {
+                options.SingleApiVersion(new Info
+                {
+                    Version = "v1",
+                    Title = "ToDo API",
+                    Description = "A simple example ASP.NET Core Web API",
+                    TermsOfService = "None",
+                    Contact = new Contact { Name = "Shayne Boyer", Email = "", Url = "http://twitter.com/spboyer"},
+                    License = new License { Name = "Use under LICX", Url = "http://url.com" }
+                });
+
+                //determine base path for executing application
```

Comment:
Code comments need to use proper sentence style.

// Determine base path for the app.

I don't mind "executing" but it's officially banned by our editorial department- but they never review our docs so we can use it. But I don't think you really need it here, so I avoid it when I can.